test/testcafe/framework/shared/index.js

```diff
@@ -5,9 +5,22 @@ import { t } from 'testcafe';
 // 2. Widget transitions to a new page and animations have finished (afterRender)
 // 3. Context object from the 'afterReady' event 
 export async function checkConsoleMessages(context = {}) {
+  if (!Array.isArray(context)) {
```

Comment:
Shall we break down into 2 different methods?

Reply:
Another great use-case for Typescript! 😄